packages/data-service-generator/src/server/package-json/create-package-json.ts

```diff
@@ -12,13 +12,12 @@ import { paramCase } from "param-case";
 import { promises as fs } from "fs";
 
 const PACKAGE_JSON_ENCODING = "utf-8";
-const PACKAGE_JSON_TEMPLATE = "package.template.json";
 const PACKAGE_JSON_FILE_NAME = "package.json";
 
-const filePath = resolve(__dirname, PACKAGE_JSON_TEMPLATE);
+const filePath = resolve(__dirname, PACKAGE_JSON_FILE_NAME);
 
 export async function createServerPackageJson(): Promise<Module[]> {
-  const fileContent = await fs.readFile(filePath, "utf-8");
+  const fileContent = await fs.readFile(filePath, PACKAGE_JSON_ENCODING);
```

Comment:
Same question here - wouldn't reading a different file change the functionality?